velox/functions/prestosql/tests/ArithmeticTest.cpp

```diff
@@ -699,5 +699,32 @@ TEST_F(ArithmeticTest, truncate) {
   EXPECT_DOUBLE_EQ(truncate(123456789012345678901.23, -21).value(), 0.0);
 }
 
+TEST_F(ArithmeticTest, betaCDF) {
+  const auto betaCDF = [&](std::optional<double> a,
+                           std::optional<double> b,
+                           std::optional<double> value) {
+    return evaluateOnce<double>("beta_cdf(c0, c1, c2)", a, b, value);
+  };
+
+  EXPECT_EQ(0.09888000000000001, betaCDF(3, 4, 0.2));
```

Comment:
Please also add tests for
std::numeric_limits::infinity(), std::numeric_limits::max(), std::numeric_limits::min(), std::numeric_limits::quiet_NaN() for the different parameters.

Reply:
Done